libelement/src/instruction_tree/instructions.hpp

```diff
@@ -15,6 +15,13 @@
 
 namespace element
 {
+    //Element treats negative numbers and 0 as false
+    //todo: update to handle NAN's, once we've decided if they're truthy or falsy
```

Comment:
My gut feeling is that NaNs should be falsy, which this would behave correctly for.

Reply:
I agree :) but I'm not sure that's what craig has in mind. I'd go a step further and say negative numbers should also be falsy ;b

Reply:
Actually wait, I was thinking of it the other way around, I like the defaults in C++ 🤷🏼‍♂️ but now I'm not sure :/